docs/source/recipes/widget.md

```diff
@@ -146,14 +146,102 @@ const applyConfig = (config) => {
 Based on this setup, Volto will render this field with the `TokenWidget`.
 
 
-```{seealso}
-See [storybook](https://6.dev-docs.plone.org/storybook) with available widgets.
+## Relation fields
```

Comment:
Section headings without narrative text should be avoided. They should either:

- have introductory general text describing their sub-sections, or
- be removed and their sub-sections demoted.